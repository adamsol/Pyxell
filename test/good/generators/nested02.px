
func f(n: Int): Rat... def
    func g(n: Int): Rat def
        if n == 0 do
            return 1
        n -= 1
        return g(n) / 2

    yield g(n)
    yield n * g(n)

print [...f(0)]
print [...f(3)]
