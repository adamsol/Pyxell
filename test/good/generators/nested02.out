[1, 0]
[0.125, 0.375]
